README.md

```diff
@@ -1,21 +1,14 @@
-# Designkit Typography
+# designkit-typography
+1.1.0
 
 Sass module for typography at RightScale.
 
 ## Install
-
-```bash
-npm i designkit-typography
 ```
-
-## Usage
-
-```html
-<div class="headline f3 line-height-1">Headline Text</div>
-<div class="bold f4">Bold body text</div>
+npm i --save designkit-typography
 ```
 
-## CSS
+### CSS
```

Comment:
Is this supposed to be an H3 rather than an H2 like the other headers?

Reply:
Fixed in the main template too https://github.com/rightscale-design/designkit-project/blob/master/templates/README.md#css